src/Auctions/Auction.sol

```diff
@@ -546,7 +546,7 @@ contract Auction is Governance2Step, ReentrancyGuard {
     /// @dev Implements the take of the auction.
     function _take(
         address _from,
-        uint256 _maxAmount,
+        uint256 __maxAmount,
```

Comment:
please refer to my comment on partial takes below. if unprotected, consider only allowing "max" takes (takes for the full amount)